common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -709,7 +712,7 @@ def _render_student_view_for_items(self, context, display_items, fragment, view=
         contents = []
         for item in display_items:
             # NOTE (CCB): This seems like a hack, but I don't see a better method of determining the type/category.
```

Comment:
^ Is this old comment still needed?

Reply:
@bradenmacdonald, probably not. I can remove it while squashing the commits.